README.md

```diff
@@ -41,8 +41,14 @@ Range, Readings
 
 Start test-driven development:
 
-1. Establish quality parameters for your project: What is the maximum complexity you would allow? How much duplication would you consider unacceptable? What is the coverage you'll aim for?
-Adapt/adopt/extend the `yml` files from one of your workflow folders.
+1. Establish quality parameters for your project: 
+    1. What is the maximum complexity you would allow?  
+       These questions are really interesting.  
+       How can one even determine that without even knowing how to write the alogrithm ? Are there any metrics or is it just a guess ?  
```

Comment:
What algorithm? Software code is just another way of stating the user need - in a way that a processor can do repeatedly.
When you convey something to another human, you can explain something in a very long sentence or a few smaller ones. Keeping a limit on sentence length helps us to practice lucid writing.
Similarly, keeping a limit on function size / nesting / complexity lets us practice clean coding, discover newer and shorter ways of writing code.
Question is, what is the limit you want to explore?

Reply:
algorithm - By algorithm i meant the logic in my unit.

> Similarly, keeping a limit on function size / nesting / complexity lets us practice clean coding, discover newer and shorter ways of writing code

What if its not possible ?  Are we allowed to change it ?